pygrgl/clicmd/common.py

```diff
@@ -11,7 +11,7 @@ def which(exe: str, required=False) -> Optional[str]:
     except subprocess.CalledProcessError:
         result = None
     if result is None:
-        for p in sys.path:
+        for p in [*sys.path, os.getcwd()]:
```

Comment:
I think the challenge with this approach is that it will change which executables are used based on where the user is running the command from, which would probably lead to some very confusing behavior!

What about something like this?

```
THISDIR = os.path.dirname(os.path.realpath(__file__))
REPO_ROOT = os.path.join(THISDIR, "..", "..")

...

    for p in [*sys.path, REPO_ROOT]:
...
```